src/NServiceBus.AcceptanceTesting/ScenarioWithContext.cs

```diff
@@ -34,28 +35,24 @@ public async Task<TContext> Run(RunSettings settings)
             var runDescriptor = new RunDescriptor(scenarioContext);
             runDescriptor.Settings.Merge(settings);
 
+            TestExecutionContext.CurrentContext.AddRunDescriptor(runDescriptor);
             ScenarioContext.Current = scenarioContext;
```

Comment:
this could probably be removed too now as we could resolve the scenario context from the TestExecutionContext but I'm not scoping this into this PR.